### PR TITLE
MM-T564 New message bar - Own user posts a reply while scrolled up in a channel

### PR DESCRIPTION
**Pull request by josephk96**
Summary

This test checks that when a user is scrolled up in a channel and replies to a message in the channel, the 'Jump To New Messages' button does not appear.

Test case: https://automation-test-cases.vercel.app/test/MM-T562

### CODE REVIEW

File: e2e/cypress/integration/notifications/own_user_posts_reply_while_scrolled_up_spec.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        cy.clickPostCommentIcon();
+
+        // # Scroll to the top of the channel so that the 'Jump to New Messages' button would be visible (if it existed)
+        cy.get('.post-list__dynamic').scrollTo('top');
+
+        // # Post a reply in RHS
+        cy.postMessageReplyInRHS('This is a test message');
+
+        cy.getLastPostId().then((postId) => {
+            // * Verify that 'Jump to New Messages' is not visible and that the new reply exists in the channel
```
Comment (saturninoabril) [inline e2e/cypress/integration/notifications/own_user_posts_reply_while_scrolled_up_spec.js:53]: I'm not seeing verification to "the new reply exists in the channel".

File: e2e/cypress/integration/notifications/own_user_posts_reply_while_scrolled_up_spec.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            cy.get('.toast__visible').should('not.be.visible');
+            cy.get(`#postMessageText_${postId}`).should('exist');
```
Comment (saturninoabril) [inline e2e/cypress/integration/notifications/own_user_posts_reply_while_scrolled_up_spec.js:55]: Seems like this fell short in verifying the last post. Maybe,
```javascript
// # Post a reply in RHS
const message = 'This is a test message';
cy.postMessageReplyInRHS(message);
...
    // * Message gets posted in Town Square
    cy.uiWaitUntilMessagePostedIncludes(message);
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @josephk96! Please see comments.

**Review by saturninoabril — APPROVED** [review]
Thanks @josephk96, LGTM.

**Review by prapti — APPROVED** [review]
Looks good to me. Thank you @josephk96. I just have one question before I go ahead and merge it: should the test case linked in this PR's description be https://automation-test-cases.vercel.app/test/MM-T564 instead of https://automation-test-cases.vercel.app/test/MM-T562? That was the only confusion I had otherwise, the test itself looks great!

**Pull request merged**